docs/notebooks/expected_improvement.pct.py

```diff
@@ -150,12 +141,14 @@ def build_model(data):
 plot_bo_points(query_points, ax[1], num_init=num_initial_points, idx_best=arg_min_idx)
 
 # %% [markdown]
-# We can visualise the model over the objective function by plotting the mean and 95% confidence intervals of its predictive distribution.
+# We can visualise the model over the objective function by plotting the mean and 95% confidence intervals of its predictive distribution. Like with the data before, we can get the model with `.try_get_final_models()` and indexing with `OBJECTIVE`.
 
 # %%
 from util.plotting_plotly import plot_gp_plotly
 
-fig = plot_gp_plotly(model[OBJECTIVE].model, mins, maxs, grid_density=30)
+fig = plot_gp_plotly(
+    result.try_get_final_models()[OBJECTIVE].model, mins, maxs, grid_density=30
```

Comment:
yeah it's weird. It's actually a hack which I have been working to fix - I just change the function to use the `ProbabilisticModel` API rather than the underling gpflow model API. It comes under #33